data.sql

```diff
@@ -58,4 +58,46 @@ UPDATE animals SET owners_id = 1 WHERE name = 'Agumon';
+INSERT INTO specialization(species_id, vets_id) VALUES (1, 1);
+INSERT INTO specialization(species_id, vets_id) VALUES (1,3), (2,3);
+INSERT INTO specialization(species_id, vets_id) VALUES (2, 5);
+
+--insert data into visits table
+INSERT INTO visits (animals_id, vets_id, visit_date) VALUES((SELECT id from animals where name = 'Agumon'),(SELECT id from vets where name = 'william tatcher'), DATE '2020-05-24');
+
+INSERT INTO visits (animals_id, vets_id, visit_date)
+VALUES((SELECT id from animals where name = 'Agumon'),(SELECT id from vets where name = 'stephanie mendez'), DATE '2020-07-22'),
+((SELECT id from animals where name = 'Pikachu'),(SELECT id from vets where name = 'maisy smith'), DATE '2020-01-05'),
+((SELECT id from animals where name = 'Pikachu'),(SELECT id from vets where name = 'maisy smith'), DATE '2020-03-08'),
+((SELECT id from animals where name = 'Pikachu'),(SELECT id from vets where name = 'maisy smith'), DATE '2020-05-14'),
+((SELECT id from animals where name = 'Gabumon'),(SELECT id from vets where name = 'jack harkness'), DATE '2021-02-02');
+
+INSERT INTO visits (animals_id, vets_id, visit_date)
+VALUES((SELECT id from animals where name = 'Devimon'),(SELECT id from vets where name = 'stephanie mendez'), DATE '2021-05-04'),
+((SELECT id from animals where name = 'plantom'),(SELECT id from vets where name = 'maisy smith'), DATE '2019-12-21'),
+((SELECT id from animals where name = 'plantom'),(SELECT id from vets where name = 'maisy smith'), DATE '2021-04-07'),
+((SELECT id from animals where name = 'plantom'),(SELECT id from vets where name = 'william tatcher'), DATE '2020-08-10'),
+((SELECT id from animals where name = 'charmander'),(SELECT id from vets where name = 'jack harkness'), DATE '2021-02-24');
+
+INSERT INTO visits (animals_id, vets_id, visit_date)
+VALUES((SELECT id from animals where name = 'squirtle'),(SELECT id from vets where name = 'stephanie mendez'), DATE '2019-09-29'),
+((SELECT id from animals where name = 'blossom'),(SELECT id from vets where name = 'stephanie mendez'), DATE '2020-05-24'),
+((SELECT id from animals where name = 'blossom'),(SELECT id from vets where name = 'william tatcher'), DATE '2021-01-11'),
+((SELECT id from animals where name = 'angemon'),(SELECT id from vets where name = 'jack harkness'), DATE '2020-10-03'),
+((SELECT id from animals where name = 'angemon'),(SELECT id from vets where name = 'jack harkness'), DATE '2020-11-04');
+
+INSERT INTO visits (animals_id, vets_id, visit_date)
+VALUES((SELECT id from animals where name = 'boarmon'),(SELECT id from vets where name = 'maisy smith'), DATE '2019-12-21'),
+((SELECT id from animals where name = 'boarmon'),(SELECT id from vets where name = 'maisy smith'), DATE '2019-04-07'),
+((SELECT id from animals where name = 'boarmon'),(SELECT id from vets where name = 'maisy smith'), DATE '2020-08-10'),
+((SELECT id from animals where name = 'boarmon'),(SELECT id from vets where name = 'maisy smith'), DATE '2020-02-24');
```

Comment:
- [ ] Your queries are fair enough but I noticed that some of your data information are different from the date of your project requirement for example **DATE**  😢  I would like to strongly recommend you read the project requirement and make sure that the data are the same as defined on it. Please make sure to correct them and pay attention to details 👍